website/docs/components/form/primitives/index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Form Primitives
+title: Primitives
```

Comment:
you'll need to update the illustration's filename to `primitives.png`

Reply:
See https://github.com/hashicorp/design-system/pull/994